### PR TITLE
Adds 6 new bottles of booze, 20+ cocktails

### PR DESCRIPTION
**Pull request by eponymousmallard**
Giving the bar some completely necessary totally non-bloaty love. 6 new types of booze - Triple Sec, Blue Curacao, Bitters and Schnapps, some "premium" champagne that isn't actually different from its parent at all, and cinnamon-flavored whisky. Along with that, fourteen cocktails cherrypicked from the IBA's list, and eight cocktails made with the cinnamon whisky. Oh, and at the behest of some anon, I gave an unused iced tea sprite a bit of a facelift to look less solid + redirected our current iced tea drink to use that sprite instead of the half-assed brown blob it was using. Booze for the booze god, bloat for the bloat throne. 
Other things to note: 
- The Mojito sprite and the blue curacao glass were there when I got here so credit for those goes to whatever dusty old spriter from ancient times did those. The iced tea glass I ruined/facelifted also isn't mine. 
- Tested in-game, everything seems to work the way it's supposed to unless I've missed something glaringly obvious. 
- In-hands for the bottles are incredibly crusty, but they exist and are aligned vaguely correctly. 

finaldrinklist
All the cocktails + booze bottles, in-game.
crustyinhands
A few of the crusty inhand sprites. 
vendingplusdispenser
The new bottles in the vendor + the additions to the dispenser (not including champagne.) Names subject to change? 
bigblockofbrown
Halfassed iced tea sprite that's no longer being used.
SHITINAGLASS
Pre-facelift/ruining iced tea glass that wasn't being used for some reason.
that'snothowiceworks
'New' iced tea glass. 
boozebottlesglassesDMI
All the bottles + their respective glasses in drinks.dmi
cocktailsDMI
All the cocktails added into drinks.dmi
ghostmojito
The mojito, since it wasn't in the previous picture.
tldr; More drinks. Tested, they work, and I have absolutely no clue if I've missed anything important or if this qualifies as atomic (a thread coder told me it was a-okay) but I'm sure someone will yell at me if I have/it isn't.
-->
:cl:
 * rscadd: Adds new booze to the boozeomat/booze dispenser and new cocktails to accompany them.
 * rscadd: Gave iced tea a newer sprite that isn't ass.

### CODE REVIEW

**Comment by eponymousmallard** [comment]
I could do that if people'd rather the reactions just worked via cinnamon + whiskey + whatever, mostly added the snowflakey whisky out of personal appeal and because it makes it a little easier to make all that stuff. Then again, I've basically never seen people grow cinnamon trees, so maybe that would be good incentive for people to start.

**Comment by strawberrytan** [comment]
cinnamon is a reagent, which could cut down on having whiskey and cinnamon whiskey be separate things in the booze dispenser

**Comment by SonixApache** [comment]
cute

**Comment by Durgen** [comment]
the fireball reagent should make the fireball spell more powerful

**Comment by Probe1** [comment]
Good on you for seeing the in hand sprites to the end and getting it all done.

**Review by Probe1 — APPROVED** [review]
Looked over, looks good.

**Comment by Zth--** [comment]
Those sprites look very good

**Comment by GeneralVeers25** [comment]
This is an incredible PR and I finally intend to try out bartender now that I can make drinks worth consuming.

**Comment by brndd** [comment]
If you've already suffered spriting in-hands for these, might you suffer a bit more and sprite inhands for the sake bottle?

**Comment by Probe1** [comment]
Make sure to update the wiki.
